schemas/dbt/run-results/v6.json

```diff
@@ -55,7 +55,8 @@
                   "success",
                   "error",
                   "skipped",
-                  "partial success"
+                  "partial success",
+                  "no-op"
```

Comment:
I thought this had already been merged as I see it here: https://schemas.getdbt.com/dbt/run-results/v6.json